Sources/RegexBuilder/CharacterClass.swift

```diff
@@ -37,33 +42,45 @@ extension CharacterClass: RegexComponent {
 
 @available(SwiftStdlib 5.7, *)
 extension CharacterClass {
+  /// A character class that matches any character that does not match this
+  /// character class.
   public var inverted: CharacterClass {
     CharacterClass(ccc.inverted)
   }
 }
 
 @available(SwiftStdlib 5.7, *)
 extension RegexComponent where Self == CharacterClass {
+  /// A character class that matches any element.
+  ///
+  /// This character class is unaffected by the `dotMatchesNewlines()` method.
   public static var any: CharacterClass {
     .init(DSLTree.CustomCharacterClass(members: [.atom(.any)]))
   }
 
+  /// A character class that matches any element that isn't a newline.
```

Comment:
If this is `.`, good to call out, similarly for any of these that have direct literal analogues.